java/core/src/main/java/sleeper/core/deploy/ClientJar.java

```diff
@@ -0,0 +1,121 @@
+/*
+ * Copyright 2022-2025 Crown Copyright
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package sleeper.core.deploy;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+/**
+ * Definitions of jar files used to deploy client functions.
```

Comment:
I'm not sure what it means to "deploy client functions". The clients jar is used by scripts, including the admin client. The bulk import runner jar is used when we run bulk import jobs on EMR Serverless. The CDK jar is used when we invoke the CDK to deploy instances of Sleeper.